Having sqlalchemy installed breaks freezegun

So I get this:

`___________________ TestNotifications.test_conference_start ____________________
tests_python/api/test_notifications.py:20: in test_conference_start
    freez.start()
env/lib/python3.4/site-packages/freezegun/api.py:250: in start
    if attribute_value == real_datetime:
env/lib/python3.4/site-packages/sqlalchemy/orm/path_registry.py:54: in __eq__
    self.path == other.path
E   AttributeError: type object 'datetime.datetime' has no attribute 'path'`

Out of this code:

`freez = freeze_time("2017-01-14 12:00:00")
        freez.start()`

This worked in 0.2.2.

--- a/freezegun/api.py
+++ b/freezegun/api.py
@@ -244,15 +244,19 @@ class _freeze_time(object):
                     continue
                 try:
                     attribute_value = getattr(module, module_attribute)
-                except ImportError:
-                    # For certain libraries, this can result in ImportError(_winreg)
+                except (ImportError, AttributeError):
+                    # For certain libraries, this can result in ImportError(_winreg) or AttributeError (celery)
                     continue
-                if attribute_value == real_datetime:
-                    setattr(module, module_attribute, FakeDatetime)
-                if attribute_value == real_date:
-                    setattr(module, module_attribute, FakeDate)
-                if attribute_value == real_time:
-                    setattr(module, module_attribute, fake_time)
+                try:
+                    if attribute_value == real_datetime:
+                        setattr(module, module_attribute, FakeDatetime)
+                    if attribute_value == real_date:
+                        setattr(module, module_attribute, FakeDate)
+                    if attribute_value == real_time:
+                        setattr(module, module_attribute, fake_time)
+                except AttributeError:
+                    # If it's not possible to compare the value to real_XXX (e.g. hiredis.version)
+                    pass
 
         datetime.datetime.times_to_freeze.append(self.time_to_freeze)
         datetime.datetime.tz_offsets.append(self.tz_offset)
@@ -287,15 +291,18 @@ class _freeze_time(object):
                     continue
                 try:
                     attribute_value = getattr(module, module_attribute)
-                except ImportError:
+                except (ImportError, AttributeError):
                     # For certain libraries, this can result in ImportError(_winreg)
                     continue
-                if attribute_value == FakeDatetime:
-                    setattr(module, module_attribute, real_datetime)
-                if attribute_value == FakeDate:
-                    setattr(module, module_attribute, real_date)
-                if isinstance(attribute_value, FakeTime):
-                    setattr(module, module_attribute, real_time)
+                try:
+                    if attribute_value == FakeDatetime:
+                        setattr(module, module_attribute, real_datetime)
+                    if attribute_value == FakeDate:
+                        setattr(module, module_attribute, real_date)
+                    if isinstance(attribute_value, FakeTime):
+                        setattr(module, module_attribute, real_time)
+                except AttributeError:
+                    pass
 
     def decorate_callable(self, func):
         def wrapper(*args, **kwargs):

